v2/contribute/decisions/oauth/0011-frontend-routes.mdx

```diff
@@ -0,0 +1,48 @@
+---
+id: "0011"
+title: Redirect to the standard login route if required
+hide_title: true
+---
+
+import DecisionHeader from "/src/components/decisionLogs/DecisionHeader"
+import ArgumentPro from "/src/components/decisionLogs/ArgumentPro"
+import ArgumentCon from "/src/components/decisionLogs/ArgumentCon"
+import ArgumentNeut from "/src/components/decisionLogs/ArgumentNeut"
+
+# Redirect to the standard login route if required
+
+<DecisionHeader status="proposed" lastUpdate="2023-05-11" created="2023-05-11" deciders={["rishabhpoddar", "porcellus"]} proposedBy={["porcellus"]} />
+
+## Context and Problem Statement
+
+When the user redirected to the app during the OAuth2 sign-in process, there are a few routes we can choose to provide.
+
+## Considered Options
+
+* Merge functionality into existing auth page
+* Add new route for OAuth2 specific authentication that embeds the sign in/up components
+* Add new route that redirects to default auth page
+
+## Decision Outcome
+
```

Comment:
Add that this route also is redirected back to from the auth page post login and then this route will redirect the user back to the client app